src/components/customizableAside.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Image from 'next/image';
+import styles from './customizableAside.module.css';
+
+function CustomizableAside() {
```

Comment:
@Sheyitofunmi I think because this is a customizable aside, it should be customizable by passing in props for the following attributes:
- the value for `className`
- the value for `src`
- the values for `width`, `height` and `alt`
- the content of `<h2>` (with max length)
- the content of the `<p>` tag (with max length)

This is so that, when I call this component, I call it like:

```js
<CustomizableAside
   className={styles.some.css.file}
   src={"path/to/img"}
   width={110}
   height={110}
   alt={"avatar"}
   h2Content={"Lorem ipsum lorem ipsum"}
   pContent={"lorem ipsum et al et al"}
/>
```

This is an actually customizable CustomizableAside, what do you think?

Reply:
i think this is perfect, i am going to go for it

Reply:
I think this is a good suggestion. I think that this should be the case for all the customizable components